helm-charts/charts/kube-starrocks/values.yaml

```diff
@@ -15,10 +15,15 @@ global:
 # the starrockscluster crd' name.
 nameOverride: ""
 # This configuration is used to modify the root password during initial deployment.
-# After deployment is completed, modify the password here and upgrade, does not modify the running root password
+# After deployment is completed, modify the password here and upgrade, does not modify the running root password.
+# It also supports providing secret names that contain passwords, using the password in the secret instead of the plaintext in the value.xml.
```

Comment:
```suggestion
# It also supports providing secret name that contains password, using the password in the secret instead of the plaintext in the values.yaml.
```

Reply:
done